website/cypress/e2e/constants.ts

```diff
@@ -0,0 +1,25 @@
+const PORT = '8080';
```

Comment:
just noticed where this file is, please move it to fixtures or directly into the cypress folder